### DISCUSSION TITLE
编码问题

### DISCUSSION THREAD

**Discussion by liaosunny123** [Mrs4s/go-cqhttp#909] [Q&A]
通过CMD导出内容编码有问题，具体表现为中文乱码？请问GO有规定编码吗
cmd启动时为正常的文字，但是启动Go后中文全部乱码

**Comment by Ink-33** [comment]
urf8

**Reply by liaosunny123** [reply]
使用chcp 65001好像并不能解决

**Reply by Ink-33** [reply]
请您自行在stack overflow等网站寻求解决方案

**Reply by Ink-33** [reply]
或者换用powershell
